ui/src/reducers/index.js

```diff
@@ -19,13 +19,18 @@ import {
   UPDATE_SP_RESOURCE,
   UPDATE_SUBPOLICY_RESOURCE,
   UPDATE_SP_META,
-  DELETE_SUBPOLICY
+  DELETE_SUBPOLICY,
+  UPDATE_POSSIBLE_DEFAULTS_LIST
 } from "../actions";
 
-// TODO: change to fetch fn that gets from nopas backend
+export const possibleDefaults = {
+  subpolicies: ["CoreRatio"],
+  ensemblers: ["Conservative"]
```

Comment:
Use the consts that you defined

Reply:
Haha okay paiseh such constants don't exist; my mind melded the Go code with your frontend code.